src/ray/gcs/gcs_server/gcs_placement_group_manager.cc

```diff
@@ -241,31 +241,25 @@ void GcsPlacementGroupManager::OnPlacementGroupCreationFailed(
       // NOTE: If a node is dead, the placement group scheduler should try to recover the
       // group by rescheduling the bundles of the dead node. This should have higher
       // priority than trying to place other placement groups.
-      pending_placement_groups_.emplace_front(std::move(placement_group));
+      AddToPendingQueue(std::move(placement_group), 0);
```

Comment:
0 is confusing here. Can you write the argumenet name in the code like this? 

```suggestion
      AddToPendingQueue(std::move(placement_group), /*[argument_name]*/0);
```